README.md

```diff
@@ -1,5 +1,7 @@
 # FFI::Aspell
 
+[![Build Status](https://travis-ci.org/YorickPeterse/ffi-aspell.svg?branch=master)](https://travis-ci.org/YorickPeterse/ffi-aspell)
+
```

Comment:
It's fairly common to put the Travis badge in the README, mainly so it's a quick indicator if tests are running and that they're green.

https://github.com/search?q=%5B%21%5BBuild+Status%5D%28https%3A%2F%2Ftravis-ci.org%2F&ref=searchresults&type=Code&utf8=%E2%9C%93

You sure you want to remove?